hydra-cluster/src/Hydra/Cluster/Scenarios.hs

```diff
@@ -1096,6 +1096,48 @@ threeNodesNoErrorsOnOpen tracer tmpDir node@RunningNode{nodeSocket} hydraScripts
       Right _headIsOpen ->
         pure ()
 
+-- | Hydra nodes ABC run on ABC cluster and connect to each other.
+-- Hydra nodes BC shut down.
+-- Hydra nodes BC run on BC cluster and connect to each other.
+-- Hydra nodes BC shut down.
+-- Hydra nodes BC run and connect ABC cluster again.
+nodeCanSupportMultipleEtcdClusters :: Tracer IO EndToEndLog -> FilePath -> RunningNode -> [TxId] -> IO ()
+nodeCanSupportMultipleEtcdClusters tracer workDir RunningNode{networkId, nodeSocket} hydraScriptsTxId = do
```

Comment:
Is this test now failing on `master`, but passing on this branch? Can you show me that somehow or do I need to pick this change to see it fail?

Reply:
Pushed a revert on top. Let's see.